tests/nightly/nightly-autorun.sh

```diff
@@ -16,9 +16,13 @@
 
 # Get the latest code from vmware/vic repo
 cd ~/go/src/github.com/vmware/vic
-git checkout master
+git checkout -f master
 git remote update
 git rebase upstream/master
+git checkout -f ui-nightlytest/ui-tests \
+    tests/resources/VCH-Util.robot \
+    ui/vic-uia \
+    ui/vic-ui-h5c/uia
```

Comment:
Are you using this to throw away local changes that happen with the build process?  I think git reset might be a better idea?